### PR TITLE
Rosnode list

### PR DESCRIPTION
**Pull request by Karsten1987**
Connect to ros2/cli_tools#1

### CODE REVIEW

File: rcl/test/CMakeLists.txt [hunk 1]
```diff
@@ -63,6 +63,17 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation}
   )
 
+  rcl_add_custom_gtest(test_get_node_names${target_suffix}
+    SRCS rcl/test_get_node_names.cpp
+    ENV
+      ${extra_test_env}
+      EMPTY_TEST=
+      NORMAL_TEST=foo
```
Comment (dirk-thomas) [inline rcl/test/CMakeLists.txt]: What are these two environment variables for? They don't seem to be used in this patch.

Comment (mikaelarguedas) [inline rcl/test/CMakeLists.txt]: looks like a copy-paste artifact and should be removed. these are used to [test getenv in common .c](https://github.com/ros2/rcl/blob/d3a27df6e2dda9638acf2d0e57eefeb22197742f/rcl/test/rcl/test_common.cpp#L29-L37) and are not used for the test_get_node_names test

Comment (dirk-thomas) [inline rcl/test/CMakeLists.txt]: Another unrelated question: why are these tests run for each rmw implementation?

Comment (Karsten1987) [inline rcl/test/CMakeLists.txt]: clearly copy&paste error. I'll open another PR to correct this.

**Pull request merged**